analyses/snv-callers/README.md

```diff
@@ -206,38 +267,46 @@ In order to compare mutations across callers, I created a `mutation_id` from com
 
 If mutation_id's are identical among MAF files, they are considered the same.
 
+### Consensus mutation call
+
+After the comparisons amongst the callers, VarDict proved to be too unreliable and called low VAF mutations.
+Moving forward, mutations that were identified by Lancet, Mutect2 and Strelka2 were included in the final list of mutations for each sample.
+The consensus mutations themselves are saved to a MAF-like file `consensus_mutation.maf.tsv.zip` to the `consensus`.
+It is being called a "MAF-like" file because it has many of the same fields as a MAF file but..  
+  - Does not contain the version string in the first row   
+  - Has extraneous annotation data has been removed (columns with all `NA`s)  
+  - Has VAF calculations and other variables that are calculated by the [`set_up_maf` function](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/util/wrangle_functions.R#L11).
+
 ## Overall file structure
 ```
 OpenPBTA-analysis
 ├── analyses
 │   └── snv-callers
-│       ├── run_variant_caller_reports.R
+│       ├── run_caller_analysis.sh
+│       ├── compare_snv_callers.Rmd
 │       ├── scripts
 │       │   ├── 00-set-up.R
 │       │   ├── 01-calculate_vaf_tmb.R
 │       │   └── 02-run_eval.R
 │       ├── util
 │       │    ├── plot_functions.R
 │       │    └── wrangle_functions.R
-│       ├── bed_regions
-│       │   ├── wxs_bed_regions_all.tsv
-│       │   ├── lancet_wgs_bed_regions.tsv
-│       │   ├── mutect2_wgs_bed_regions.tsv
-│       │   ├── strelka2_wgs_bed_regions.tsv
-│       │   └── vardict_wgs_bed_regions.tsv
 │       ├── results
+│       │   ├── consensus
+│       │   │   ├── consensus_mutation_tmb.tsv
+│       │   │   └── consensus_mutation.maf.tsv
 │       │   ├── lancet
-│       │   │   ├── lancet_vaf.tsv
-│       │   │   ├── lancet_vaf.tsv.zip
-│       │   │   ├── lancet_tmb.tsv
-│       │   │   ├── lancet_tmb.tsv.zip
-│       │   │   ├── lancet_region.tsv
-│       │   │   ├── lancet_region.tsv.zip
+│       │   │   ├── lancet_vaf.rds
+│       │   │   ├── lancet_vaf.rds.zip
```

Comment:
Do you have both the file and a zipped version? (I would assume the .rds is compressed or not internally)

Reply:
You're right. This is redundant/inaccurate.